ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -400,6 +400,7 @@ static const uint32_t symm_key_boolprops[] = {
 static const uint32_t symm_key_opt_or_null[] = {
 	PKCS11_CKA_WRAP_TEMPLATE, PKCS11_CKA_UNWRAP_TEMPLATE,
 	PKCS11_CKA_DERIVE_TEMPLATE, PKCS11_CKA_VALUE,
+	PKCS11_CKA_CHECK_VALUE,
```

Comment:
should move to `symm_key_optional` below.
It will prevent you from removing the attribute when present.